src/js/controllers/Controller.js

```diff
@@ -0,0 +1,51 @@
+import { View } from '../view/View.js';
+import { LottoGame } from '../model/LottoGame.js';
+import { validator } from '../utils.js';
+
+export class Controller {
+  constructor() {
+    this.view = new View();
+    this.lottoGame = new LottoGame();
+    this.purchaseLotto();
+  }
+
+  purchaseLotto() {
+    this.view.purchaseBtn.addEventListener('click', (e) => {
+      e.preventDefault();
+
+      if (this.detectInvalidInput()) {
+        return;
+      }
+      this.lottoGame.insertMoney(Number(this.view.moneyInput.value));
+      this.lottoGame.buyLotto();
+
+      this.view.uncheckToggleSwitch();
+      this.view.clearMoneyInput();
+      this.view.showLottoStatusContainer();
+      this.view.showWinningLottoContainer();
+      this.view.showPurchasedLottos(this.lottoGame.lottoWallet);
+
+      this.bindToggleEvent();
+    });
+  }
+
+  bindToggleEvent() {
+    this.view.toggleBtn.addEventListener('click', () => {
+      if (this.view.toggleBtn.checked) {
+        this.view.lottosToggleOn(this.lottoGame.lottoWallet);
+        return;
+      }
+      this.view.lottosToggleOff(this.lottoGame.lottoWallet);
```

Comment:
```suggestion
      this.view.[this.view.toggleBtn.checked ? "lottosToggleOn" : "lottosToggleOff"](this.lottoGame.lottoWallet);
```
3항연산자로도 작성해 볼 수 있겠네요.

Reply:
네! 저도 if문이 더 가독성이 좋다고 생각해요~! 😀